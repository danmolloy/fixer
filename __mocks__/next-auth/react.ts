import { mockSession } from '../session';

module.exports = {
  __esModule: true,
  ...jest.requireActual('next-auth/react'),
  auth: jest.fn(() => {
    return mockSession;
  }),
  signIn: jest.fn(),
  signOut: jest.fn(),
  useSession: jest.fn(),
};
