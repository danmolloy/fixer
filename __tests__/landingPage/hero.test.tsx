import { render, screen } from "@testing-library/react"
import "@testing-library/jest-dom"
import React from "react";
import Hero from "../../components/externalSite/landingPage/hero";

describe("Hero component", () => {
  beforeEach(() => {
    render(<Hero />)
  })
  it("Renders", () => {
    const heroDiv = screen.getByTestId("hero-div")
    expect(heroDiv).toBeInTheDocument()
  })
  it("Matches snapshot", () => {
    const heroDiv = screen.getByTestId("hero-div")
    expect(heroDiv).toMatchSnapshot()
  })
})