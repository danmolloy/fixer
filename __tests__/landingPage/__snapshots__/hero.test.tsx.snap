// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component Matches snapshot 1`] = `
<div
  class=" flex flex-col items-center text-center px-10 py-32"
  data-testid="hero-div"
>
  <h1
    class="text-6xl font-semibold py-4"
  >
    Communication
     
    <span
      class="text-blue-600"
    >
      made simple
    </span>
     
    for fixers and musicians.
  </h1>
  <p
    class="p-4 text-slate-700"
  >
    Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium.
  </p>
  <div>
    <button
      class="m-2 bg-black hover:bg-slate-700 text-white rounded-full py-2 px-4 text-sm "
    >
      Get 6 months free
    </button>
    <button
      class="m-2 border hover:border-slate-300 text-black rounded-full py-2 px-4 text-sm "
    >
      Watch video
    </button>
  </div>
</div>
`;
